src/equisolve/vanilla/preprocessing/_base.py

```diff
@@ -0,0 +1,83 @@
+import numpy as np
+
+from equistore import TensorMap, Labels
+from equistore.block import TensorBlock, Gradient
+
+def ase_frames_to_properties_tensormap(frames, get_energy, get_forces=None, get_viral_stress=None):
```

Comment:
Could `get_energy` not also `None`. I mean I can imagine a system where I only have forces.

Reply:
I have considered this in the newest commit, but have not yet implemented it, because I don't know how this will work with TensorMap.